src/player-loader.js

```diff
@@ -18,6 +18,7 @@ class PlayerLoader {
       castMediaElement.style.position = 'absolute';
       const playerContainerEl = Utils.Dom.createElement('div');
       playerContainerEl.id = PLAYER_CONTAINER;
+      playerContainerEl.style.display = 'block';
```

Comment:
lets add a comment here so it is clear.
Does it change anything for regular chromecast devices?

Reply:
done, and no